### PR TITLE
test: Initial work for workspace-type integration tests

### PR DESCRIPTION
**Pull request by ealemayehu**
1. Implemented workspace-types resources
2. Implement integration test for get API

### CODE REVIEW

File: main/integration-tests/support/resources/workspace-types/workspace-types.js
Comment (hatimkhan) [inline main/integration-tests/support/resources/workspace-types/workspace-types.js]: instead of this.setup.settings,  can you use this.settings  (the CollectionResource exposes this.settings for you).

File: main/integration-tests/__test__/api-tests/workspace-types/get-workspace-type.test.js
Comment (hatimkhan) [inline main/integration-tests/__test__/api-tests/workspace-types/get-workspace-type.test.js]: In most of our tests description, I don't think we say "should pass", maybe we can do something like "should return approved types even if user is not an admin"

File: main/integration-tests/__test__/api-tests/workspace-types/get-workspace-type.test.js
Comment (hatimkhan) [inline main/integration-tests/__test__/api-tests/workspace-types/get-workspace-type.test.js]: Similar to my comments above, I think the description is a bit hard to read.

File: main/integration-tests/__test__/api-tests/workspace-types/get-workspace-type.test.js
Comment (hatimkhan) [inline main/integration-tests/__test__/api-tests/workspace-types/get-workspace-type.test.js:1]: I suggest that we add tests for anonymous sessions. There are a few places were we did that, such as the user and study tests.

File: main/integration-tests/support/resources/workspace-types/workspace-type.js
Comment (hatimkhan) [inline main/integration-tests/support/resources/workspace-types/workspace-type.js]: I think we have been doing camel case for the type, per the second tab in the spreadsheet.

File: main/integration-tests/support/resources/workspace-types/workspace-types.js
Comment (hatimkhan) [inline main/integration-tests/support/resources/workspace-types/workspace-types.js]: I think we have been doing camel case for the types per the second tab in the spreadsheet

File: main/integration-tests/__test__/api-tests/workspace-types/get-workspace-type.test.js
Comment (ealemayehu) [reply main/integration-tests/__test__/api-tests/workspace-types/get-workspace-type.test.js]: Done

File: main/integration-tests/support/resources/workspace-types/workspace-type.js
Comment (ealemayehu) [reply main/integration-tests/support/resources/workspace-types/workspace-type.js]: Done

File: main/integration-tests/support/resources/workspace-types/workspace-types.js
Comment (ealemayehu) [reply main/integration-tests/support/resources/workspace-types/workspace-types.js]: Done

**Review by SanketD92 — DISMISSED** [review]
The initial setup required LGTM. 
(Need to add tests, which I assume will come after this)

File: main/integration-tests/support/resources/workspace-types/workspace-type.js
Comment (hatimkhan) [inline main/integration-tests/support/resources/workspace-types/workspace-type.js]: Sorry, camel case, not pascal case.

Comment (ealemayehu) [reply main/integration-tests/support/resources/workspace-types/workspace-type.js]: Done

**Review by hatimkhan — APPROVED** [review]

**Pull request merged**
